sample/src/main/java/com/funrisestudio/stepprogresssample/MainActivity.kt

```diff
@@ -24,6 +25,16 @@ class MainActivity : AppCompatActivity() {
                     Toast.LENGTH_SHORT
             ).show()
         }
+
+        vStepProgress.setNodeTitleSize(50)
```

Comment:
I'm afraid that programmatic demo would be an overhead for the sample

Reply:
fixed